### PR TITLE
WFLY-13268 MP Fault Tolerance quickstart cannot resolve dependencies …

### PR DESCRIPTION
**Pull request by rhusar**
…with wildlfy-microprofile BOM

Jira
https://issues.redhat.com/browse/WFLY-13268

Downstream
https://github.com/wildfly/quickstart/pull/402

### CODE REVIEW

File: microprofile-fault-tolerance/pom.xml [hunk 1]
```diff
@@ -52,6 +52,13 @@
 
     <dependencyManagement>
         <dependencies>
+            <dependency>
```
Comment (xstefank) [inline microprofile-fault-tolerance/pom.xml]: you don't need the whole dependencyManagement section here as both boms are in parent.

Comment (rhusar) [reply microprofile-fault-tolerance/pom.xml]: That's done towards the efforts of removing the boms from parent as they should not be there.

**Comment by rhusar** [comment]
+1 @xstefank I agree that the `widlfly-microprofile` BOM should be sufficient and we should not mix and match.

File: microprofile-fault-tolerance/pom.xml (hunk 1 above)
Comment (rhusar) [reply microprofile-fault-tolerance/pom.xml]: Resolving discussion - cc @emmartins

**Comment by emmartins** [comment]
IMHO You should explain why you importing the BOMs, e.g. XML comment before each import (in both the pom.xml and README.adoc XML snippet)

**Comment by rhusar** [comment]
@emmartins I could but this doesn't seem to be a standard, e.g. https://github.com/wildfly/quickstart/blame/master/microprofile-jwt/README.adoc#L180-L200

**Comment by emmartins** [comment]
@rhusar it's something that should be done for all quickstarts once BOMs imports are added

**Comment by rhusar** [comment]
@emmartins OK, added a note.

**Pull request merged**